dragonchain/lib/logs.py

```diff
@@ -0,0 +1,26 @@
+import os
```

Comment:
why is this file called logs.py when it's very specifically for openfaas? Should probably go in faas.py